Sources/RegexBenchmark/CLI.swift

```diff
@@ -0,0 +1,33 @@
+import ArgumentParser
+
+@main
+struct Runner: ParsableCommand {
+  @Argument(help: "Names of benchmarks to run")
+  var specificBenchmarks: [String] = []
+  
+  @Option(help: "Run only once for profiling purposes")
+  var profile = false
+  
+  @Option(help: "How many samples to collect for each benchmark")
+  var samples = 20
+    
+  func makeRunner() -> BenchmarkRunner {
+    var benchmark = BenchmarkRunner("RegexBench", samples)
+    benchmark.addReluctantQuant()
+    benchmark.addBacktracking()
+    benchmark.addCSS()
+    benchmark.addFirstMatch()
+    return benchmark
```

Comment:
Could we make it an array or some other way to make registration easier? Seems like we could then filter that array when creating the runner